src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/Arrival.kt

```diff
@@ -38,8 +38,34 @@ data class Arrival(
   @Schema(description = "Is the arrival in custody according to NOMIS", example = "true")
   val isCurrentPrisoner: Boolean = false,
 
-  @Schema(description = "Gender of the arrival according to Book a Secure Move. Only returns MALE or FEMALE. If Book a Secure Move returns Trans, this resource will return null as we will need to capture legal gender information", example = "MALE")
-  val gender: Gender? = null
+  @Schema(
+    description = "Gender of the arrival according to Book a Secure Move. Only returns MALE or FEMALE. If Book a Secure Move returns Trans, this resource will return null as we will need to capture legal gender information",
+    example = "MALE"
+  )
+  val gender: Gender? = null,
+
+  @Schema(
+    description = "A list of all potential matches",
+    example = "[firstName: \"Sam\",lastName: \"Smith\",dateOfBirth: \"1971-02-01\",prisonNumber: \"A1234BC\",pncNumber: \"01/1234X\"]"
```

Comment:
the example isn't needed here and will generally be generated by the examples you have on the Potential matches below.